src/policies/security.ts

```diff
@@ -0,0 +1,233 @@
+import { a } from "pepr";
+
+import { When, containers } from "./common";
+import { exemptDropAllCapabilities, exemptSELinuxTypes } from "./exemptions/security";
+import { V1SecurityContext } from "@kubernetes/client-node";
+
+/**
+ * This policy ensures that Pods do not allow privilege escalation.
+ *
+ * The `allowPrivilegeEscalation` field in a container's security context should either be undefined
+ * or set to `false` to prevent potential security vulnerabilities.
+ *
+ * Running containers in privileged mode disables many security mechanisms and grants extensive
+ * access to host resources, which can lead to security breaches.
+ *
+ * @related https://repo1.dso.mil/big-bang/product/packages/kyverno-policies/-/blob/main/chart/templates/disallow-privilege-escalation.yaml
+ * @related https://repo1.dso.mil/big-bang/product/packages/kyverno-policies/-/blob/main/chart/templates/disallow-privileged-containers.yaml
+ */
+When(a.Pod)
+  .IsCreatedOrUpdated()
+  .Validate(request => {
+    const hasPrivilegedContainer = containers(request)
+      .flatMap(c => c.securityContext || {})
+      .find(ctx => ctx.allowPrivilegeEscalation || ctx.privileged);
+
+    if (hasPrivilegedContainer) {
+      return request.Deny(
+        "Privilege escalation is disallowed. The allowPrivilegeEscalation and privileged fields must be undefined or set to `false`.",
```

Comment:
UX nit: I think there's a mix of `must not be set to true` (example `hostNetwork` policy) and `must be undefined or set to false` would probably be good to standardize on one (my preference is the second).